### DISCUSSION TITLE
Shelly duo refuses to flash mgos to tasmota

### DISCUSSION THREAD

**Discussion by hansa031** [arendst/Tasmota#22325] [Q&A] [answered]
I have a Shelly duo with firmware 20230913-111821/v1.14.0-gcb84623 and when i do a: http://<ip>/ota?url=http://dl.dasker.eu/firmware/mg2tasmota-ShellyDuo.zip it show :
status | "updating"
has_update | false
new_version | "20230913-111821/v1.14.0-gcb84623"
old_version | "20230913-111821/v1.14.0-gcb84623"
beta_version | "20231107-162700/v1.14.1-rc1-g0617c15"

but it does not the flash the mgos to tasmota firmware.

When I do http://<ip>/ota?url=http://firmware.shelly.cloud/gen1/rc/SHBDUO-1.zip it does flash to the beta version of the Shelly firmware.

I also submitted this at mgos
Please help me. Thanks!

Grt,
Hans

**Comment by Jason2866** [comment, marked as answer]
Use the Tasmota repo https://github.com/arendst/mgos-to-tasmota

**Reply by hansa031** [reply]
Yessssssssssss!!!! This works thx!
